src/test/kotlin/no/skatteetaten/aurora/gobo/resolvers/ApplicationDeploymentsAndImagesResolversTest.kt

```diff
@@ -17,14 +18,14 @@ class ApplicationDeploymentsAndImagesResolversTest : GraphQLTestWithDbhAndSkap()
     private lateinit var applicationDeploymentsAndImages: Resource
 
     @MockkBean
-    private lateinit var applicationService: ApplicationServiceBlocking
+    private lateinit var applicationService: ApplicationService
 
     @MockkBean
     private lateinit var imageRegistryService: ImageRegistryServiceBlocking
 
     @BeforeEach
     fun setUp() {
-        every { applicationService.getApplicationDeployment(any<String>()) } returns ApplicationDeploymentResourceBuilder().build()
+        coEvery { applicationService.getApplicationDeployment(any<String>()) } returns ApplicationDeploymentResourceBuilder().build()
```

Comment:
Har det noe betydning om man bruker returns eller coAnswers på en coEvery?

Reply:
Nei, returns i en coEvery setting vil bli riktig. returns er en returVerdi og answers er en funksjon. Ganske sikker på at begge to i denne settingen vil fungere

Reply:
Den er god :ok_hand: